common-pages/src/ValueList/ByValueList.tsx

```diff
@@ -109,19 +109,19 @@ function ByValueListGroup(props: IProps)  {
                    cols={[
                        { key: 'Name', field: 'Name', label: 'Name', headerStyle: { width: '15%' }, rowStyle: { width: '15%' } },
                        { key: 'Description', field: 'Description', label: 'Description/Comments', headerStyle: { width: 'auto' }, rowStyle: { width: 'auto' } },
-                       { key: 'Items', field: 'Items', label: 'Items', headerStyle: { width: '10%' }, rowStyle: { width: '10%' }, content: (item) => items.filter(i => i.GroupID == item.ID).length },
+                       { key: 'Items', field: 'Items', label: 'Items', headerStyle: { width: '10%' }, rowStyle: { width: '10%' }, content: (item) => items.filter(i => i.GroupID === item.ID).length },
                        { key: 'Scroll', label: '', headerStyle: { width: 17, padding: 0 }, rowStyle: { width: 0, padding: 0 } },
                    ]}
                    tableClass="table table-hover"
                    data={data}
                    sortKey={sortKey}
                    ascending={asc}
                    onSort={(d) => {
-										 if (d.colKey == 'remove' || d.colKey == 'scroll' || d.colField === undefined)
+										 if (d.colKey === 'remove' || d.colKey === 'scroll' || d.colField === undefined)
 											 return;
 										 setSortKey(d.colField);
-										 if (d.colField == sortKey)
-											 setASC((asc) => !asc)
+										 if (d.colField === sortKey)
+											 setASC(false)
```

Comment:
needs to be !asc because if asc is False this will now return false instead of true. try renaming asc to d